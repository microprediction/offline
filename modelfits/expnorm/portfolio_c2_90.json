{"g1": 0.10782113404368283, "g2": 0.045862848650546795, "logK": -1.081933644177666, "loc": 0.058475172433460565, "logScale": 0.8958412089534995}