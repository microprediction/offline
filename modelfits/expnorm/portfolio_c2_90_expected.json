{"g1": 0.5072369271477145, "g2": 6.487693737660846, "logK": 0.9563247525554084, "loc": -0.1494046123560428, "logScale": -1.9653497500036605}